specification/marketplacecatalog/data-plane/Products/preview/2023-01-01-preview/products.json

```diff
@@ -0,0 +1,1818 @@
+            "default": false
+          },
+          {
+            "type": "array",
+            "name": "hideKeys",
+            "in": "query",
+            "description": "Add hide key to retrieve preview items. Hidekeys are provided in 'multi' format: &Hidekeys[]=\"first\"&Hidekeys[]=\"second\"",
+            "collectionFormat": "multi",
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "string",
+            "name": "X-API-Key",
+            "in": "header",
+            "required": true,
+            "description": "Use Api key provided by Marketplace Catalog Team as a parameter provided in the header"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns a single public or preview product",
+            "schema": {
+              "$ref": "#/definitions/ProductDetails"
+            }
+          },
+          "default": {
+            "description": " Error Responses: \n\n * 400 - In case of a misconfigured request.\n\n * 401 - In case of a missing api key header\n\n * 404 - In case product not found.\n\n * 500 - In case of internal error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/products": {
+      "get": {
+        "tags": [
+          "ListingPublicProducts"
+        ],
+        "summary": "Public Products API provides an unauthenticated endpoint for consuming commercial Marketplace public products. In addition, it enables filtering on selected product properties, search, and getting starting price information per product. Each API request must include an API key allocated to you (see X-API-Key description).Send an email to MKPL_Platform_API_DL@microsoft.com including the following details to get the API Key:\n- Customer name\n- Service/Product name\n- AD Registered App ID (if available)\n- Focal point email\n- Use case/scenario\n- Expected traffic volume, including peak requests per second (daily)",
+        "x-ms-examples": {
+          "Product_Get": {
+            "$ref": "./examples/ListingPublicProducts_Get.json"
+          }
+        },
+        "operationId": "PublicProducts_Get",
+        "parameters": [
+          {
+            "type": "string",
+            "name": "api-version",
+            "x-originalName": "apiVersion",
+            "in": "query",
+            "required": true,
+            "description": "Version of the REST API used for the request. "
+          },
+          {
+            "type": "string",
+            "name": "storefront",
+            "in": "query",
+            "description": "Relevant storefront, possible values - 'azure, amp, appsource'. Default='azure'",
+            "default": "azure"
+          },
```

Comment:
Is it array with csv?

Reply:
Storefront here can take only a single value